scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
-
-To get a list of created namespaces, run:
-
-  ```console
-  kubectl get namespaces
-  ```
-
-Tanzu Application Platform system namespaces can include:
-
-  ```console
-  - accelerator-system
-  - api-portal
-  - app-live-view
-  - app-live-view-connector
-  - app-live-view-conventions
-  - build-service
-  - cartographer-system
-  - cert-injection-webhook
-  - cert-manager
-  - conventions-system
-  - developer-conventions
-  - flux-system
-  - image-policy-system
-  - kapp-controller
-  - knative-eventing
-  - knative-serving
-  - knative-sources
-  - kpack
-  - learning-center-guided-ui
-  - learning-center-guided-w01
-  - learningcenter
-  - metadata-store
-  - scan-link-system
-  - secretgen-controller
-  - service-bindings
-  - services-toolkit
-  - source-system
-  - spring-boot-convention
-  - stacks-operator-system
-  - tanzu-cluster-essentials
-  - tanzu-package-repo-global
-  - tanzu-system-ingress
-  - tap-gui
-  - tap-install
-  - tap-telemetry
-  - tekton-pipelines
-  - triggermesh
-  ```
-
-The following is an example `ClusterImagePolicy`:
-
-  ```yaml
-  ---
-  apiVersion: signing.apps.tanzu.vmware.com/v1beta1
-  kind: ClusterImagePolicy
-  metadata:
-      name: image-policy
-  spec:
-    verification:
-      exclude:
-        resources:
-          namespaces:
-          - kube-system
-          - <TAP system namespaces>
-      keys:
-      - name: first-key
-        publicKey: |
-          -----BEGIN PUBLIC KEY-----
-          ...
-          -----END PUBLIC KEY-----
-      images:
-      - namePattern: registry.example.org/myproject/*
-        keys:
-        - name: first-key
-      - namePattern: registry.example.org/authproject/*
-        secretRef:
-          name: secret-name
-          namespace: namespace-name
-        keys:
-        - name: first-key
-  ```
+* `images`: The images block defines the patterns of images that should be
+  subject to the ClusterImagePolicy. If multiple policies match a particular
+  image, _ALL_ of those policies must be satisfied for the image to be admitted.
+  If there is no host in the `glob` field, `index.docker.io` is used for the
+  host.
 
-The `name` for the `ClusterImagePolicy` resource must be `image-policy`.
+* `authorities`: The authorities block defines the rules for discovering and
+  validating signatures. Discovery is done through the `sources` field, and can
+  be specified on any entry. Signatures are cryptographically verified using one
+  of the `key` or `keyless` fields.
 
-Add any namespaces that run container images that are not signed in the
-`spec.verification.exclude.resources.namespaces` section, such as the
-`kube-system` namespace.
+When a policy is selected to be evaluated against the matched image, the
```

Comment:
`image` -> `resolved image digest`
(because the mutation webhook resolves all images to their digest form)